### DISCUSSION TITLE
Is it possible to distinguish two variable with same data type in URL?

### DISCUSSION THREAD

**Discussion by ranvd** [pallets/flask#4565] [Q&A] [answered]
Is it possible to distinguish two variable with same data type in same URL format? For example:
`@app.route("/<string:video>")`
`@app.route("/<string:image>")`

Because, when I use `url_for()` in template calling the `download_video()` function. It will call the `change_folder()` instead. I absolutely can change the url format. Just wondering if there is any possible to apart these two.

![image](https://user-images.githubusercontent.com/48400614/166228200-8d9d11e5-5c68-4dd7-9183-53338aa99bee.png)

**Comment by davidism** [comment, marked as answer]
No, they are currently the same URL.

**Reply by ranvd** [reply]
Got it thanks.
